lib/livebook_web/live/session_live.ex

```diff
@@ -1701,6 +1702,18 @@ defmodule LivebookWeb.SessionLive do
      push_patch(socket, to: ~p"/sessions/#{socket.assigns.session.id}/settings/custom-view")}
   end
 
+  def handle_event(
+        "set_smart_cell_editor_intellisense_node",
+        %{"js_view_ref" => cell_id, "node" => node, "cookie" => _cookie},
+        socket
+      ) do
+    Session.set_cell_attributes(socket.assigns.session.pid, cell_id, %{
+      editor_intellisense_node: node
+    })
```

Comment:
I think we may as well convert to atom here, especially that we say `editor_intellisense_node: node() | nil` :)